jinahub/indexers/searcher/HnswlibSearcher/hnswlib_searcher.py

```diff
@@ -278,3 +286,19 @@ def status(self, **kwargs) -> DocumentArray:
             }
         )
         return DocumentArray([status])
+
+    @property
+    def metric_type(self):
+        metric_type = 'l2'
```

Comment:
Same here - logic needs to be explicit, this line should be an `if` statement for euclidean